src/smirnoffio.py

```diff
@@ -667,6 +653,55 @@ def build_system_mval_masks(self):
         logger.info("system_mval_masks is built for faster gradient evaluations")
         self.system_mval_masks = system_mval_masks
 
+class TorsionProfileTarget_SMIRNOFF(TorsionProfileTarget):
+    """ Force and energy matching using SMIRKS native Open Force Field (SMIRNOFF). """
+    def __init__(self,options,tgt_opts,forcefield):
+        ## Default file names for coordinates and key file.
+        self.set_option(tgt_opts,'pdb',default="conf.pdb")
+        # List of .mol2 files for SMIRNOFF to set up the system
+        self.set_option(tgt_opts,'mol2',forceprint=True)
```

Comment:
The OFF toolkit currently only "trusts" OpenEye to load mol2 files, so if this is calling the OFF toolkit, this makes OpenEye a dependency.

Reply:
Noted. If we want to use RDKit, should we load templates for individual molecules using .sdf?

Reply:
That's right. RDKit has some trouble with loading charges from SDF, but I can send you a code snippet to handle that until we work out a better solution.